app/models/manageiq/providers/kubevirt/infra_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,34 @@
+class ManageIQ::Providers::Kubevirt::InfraManager::EventCatcher::Runner < ManageIQ::Providers::BaseManager::EventCatcher::Runner
+  include ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcherMixin
+
+  ENABLED_EVENTS = {
+    'VirtualMachine'         => %w(Created Started Migrated SuccessfulCreate SuccessfulDelete ShuttingDown),
+    'VirtualMachineInstance' => %w(Created Started Migrated SuccessfulCreate SuccessfulDelete ShuttingDown)
+  }
+
+  def event_monitor_handle
+    @event_monitor_handle ||= ManageIQ::Providers::Kubevirt::InfraManager::KubernetesEventMonitor.new(@ems)
+  end
+
+  def queue_event(event)
+    event_hash = ManageIQ::Providers::Kubevirt::InfraManager::EventParser.event_to_hash(event, @cfg[:ems_id], @ems.emstype.upcase)
```

Comment:
@nasark I think we should do `@ems.class::EventParser` here so that if we were to make any Openshift specific changes we could do it in an `Openshift::InfraManager::EventParser` subclass